ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -28,6 +28,22 @@ object Cli {
     override val reads: String => Duration = Duration.parse
   }
 
+  private implicit val levelRead: Read[Level] = new Read[Level] {
```

Comment:
an alternative and less involved way of doing this would be:

```
val knownLevels = Set("ERROR", "WARN", "INFO", "DEBUG", "TRACE")
// ...
opt[String]("log-level")
      .optional()
      .validate(l => if (knownLevels.contains(l.toUpperCase) success else failure(s"Unrecogized logging level $l"))
      .action((level, c) => c.copy(logLevel = Level.valueOf(level.toUpperCase)))
```

Reply:
@gerolf-da Is this a request to change it? I do not have strong preference here. Happy to make it both ways.

Reply:
Sorry for being unclear. Yes, please change it. After that I'm 👍